docs/references/config/config.toml.md

```diff
@@ -0,0 +1,913 @@
+results from the latest height.
+```toml
+enabled = true
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+If [`grpc.laddr`](#grpcladdr) is empty, this setting is ignored and the service is not enabled.
+
+### grpc.privileged.laddr
+Configuration for privileged gRPC endpoints, which should **never** be exposed to the public internet.
+```toml
+laddr = ""
+```
+
+| Value type          | string                                             |
+|:--------------------|:---------------------------------------------------|
+| **Possible values** | TCP Stream socket (`"tcp://127.0.0.1:26658"`)      |
+|                     | Unix domain socket (`"unix:///var/run/abci.sock"`) |
+|                     | `""`                                               |
+
+If not specified, the gRPC privileged endpoints will be disabled.
+
+### grpc.privileged.pruning_service
+Configuration specifically for the gRPC pruning service, which is considered a privileged service.
+```toml
+enabled = false
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `false` |
+|                     | `true`  |
+
+Only controls whether the pruning service is accessible via the gRPC API - not whether a previously set pruning service
+retain height is honored by the node. See the [storage.pruning](#storagepruninginterval) section for control over pruning.
+
+If [`grpc.laddr`](#grpcladdr) is empty, this setting is ignored and the service is not enabled.
+
+## Peer-to-peer
+These configuration options change the behaviour of the peer-to-peer protocol.
+
+### p2p.laddr
+TCP socket address for the P2P service to listen on.
+```toml
+laddr = "tcp://0.0.0.0:26656"
+```
+
+| Value type          | string                                            |
+|:--------------------|:--------------------------------------------------|
+| **Possible values** | TCP Stream socket (`"tcp://127.0.0.1:26657"`)     |
+
+### p2p.external_address
+TCP address to use as identification with peers. Useful when the node is running on a non-routable address or when the
+node does not have the capabilities to figure out its IP address.
+```toml
+external_address = ""
+```
+
+| Value type          | string                      |
+|:--------------------|:----------------------------|
+| **Possible values** | IP:port (`"1.2.3.4:26656"`) |
+|                     | `""`                        |
+
+The port has to point to the node's P2P port.
+
+Example with a node on a non-routable network:
+- Node has local IP address `10.10.10.10` and uses port `10000` for P2P communication, set in [`p2p.laddr`](#p2pladdr).
+- The network gateway has the public IP `1.2.3.4` and we want to use publicly open port `26656` on the IP address.
+- A redirection has to be set up from `1.2.3.4:26656` to `10.10.10.10:1000`
+- The `external_address` has to be set to `1.2.3.4:26656`.
+
+### p2p.seeds
```

Comment:
Maybe this is too much detail, but the seed node is actually the last resource of addresses for a node. Namely, when the node has no address available to dial, it asks the seed node for addresses.

In this way, if there are configured persistent peers or the address book is not empty, the seed node is not really necessary. Maybe this deserves a manual regarding node discovery and connectivity.